lldb/include/lldb/Target/ThreadList.h

```diff
@@ -151,6 +151,11 @@ class ThreadList : public ThreadCollection {
   std::vector<lldb::tid_t> m_expression_tid_stack;
 
 private:
+  /// Returns true if the current ThreadPlan of any threads requested
+  /// StopOthers. This ignores OS Plugin threads that are not currently backed
+  /// by any real thread.
```

Comment:
Why is this restriction necessary?

Reply:
I suspect the authors original intent was that "if there is no backing thread, this thread is probably not going to run?". I'm not sure